src/actions/itemActions.js

```diff
@@ -103,6 +103,32 @@ export function addItem(item, itemType, authSig) {
 	}
 }
 
+export function cloneItem({ item, authSig } = {}) {
+	const newItem = { ...item }
+	return async function (dispatch) {
+		try {
+			newItem.id = ''
+			newItem.created = Date.now()
+
+			const resItem = await postAdUnit({
```

Comment:
You had to wait for the issue :)
Is this make any sense.
Why would you cone item without be able to edit it firs.
I'll make the issue.